context/build.gradle

```diff
@@ -2,5 +2,6 @@ description = 'gRPC: Context'
 
 dependencies {
     testCompile project(':grpc-testing')
+    testCompile project(':grpc-core').sourceSets.test.output
```

Comment:
`StaticTestingClassLoader` is primarily used for testing ServiceProvider which is not related to `Context`, maybe move it to testing/io.grpc.interal.testing later?